compat-data-schema.md

```diff
@@ -0,0 +1,365 @@
+
+The basic support feature represents the minimal set of functionality included
+when a feature is qualified of 'supported'. What this represents depends of the
+evolution of the feature over time, both in term of specification and of browser
+support. Another way of seeing it is to consider `"basic_support"` as representing all
+the functionality of the feature that doesn't have its own sub-feature(s).
+
+### Sub-feature
+
+A sub-feature is the basic entity having browser compatibility information. As
+explained in the previous paragraph, any feature has at least one sub-feature
+called `'basic_support'`, but it may many more.
+
+A sub-feature may have three properties.
+
+* __Sub-feature description__ contained in the `"desc"` property. It is a
+`string` that contains a human-readable description of the sub-feature. As it is
+intended to be used as a kind of caption or title for the feature, keep it short.
+the `<code>` and `<a>`, as well as the macros `{{cssxref}}`, `{{HTMLElement}}`,
+`{{htmlattrxref}}`, and `{{domxref}}` can be used. See the localization section
+below for an explanation about how this string will be localized.
+* __Compat information__ contained in the `"support"` property. It contains an
+object listing the compat information for each browser. (See the description
+below.)
+* __Status information__ contained in the `"status"` object. It contains the
+information about the stability of the sub-feature: Is it a functionality that
+is standard? Is it stable? Has it been deprecated and shouldn't be used anymore
+(See the description below.)
+
+### The `"support"` object
+Each sub-feature has support information. For each browser identifier, it
+contains a compat object with the information about versions, prefixes or
+alternate names, as well as notes.
+
+The currently accepted browser identifiers are:
+* `"webview_android"`, Webview, the former stock browser on Android,
+* `"chrome"`, Google Chrome (on desktops),
+* `"chrome_android"`, Google Chrome (on Android),
+* `"edge"`, MS Edge (on Windows),
+* `"edge_mobile"`, MS Edge, the mobile version,
+* `"firefox`", Mozilla Firefox (on desktops),
+* `"firefox_android"`, Firefox for Android, sometimes nicknamed Fennec,
+* `"ie_mobile"`, Microsoft Internet Explorer, the mobile version,
+* `"ie"`, Microsoft Internet Explorer (discontinued)
+* `"opera"`, the Opera browser (desktop), based on Blink since Opera 15,
+* `"opera_android"`, the Opera browser (Android version)
+* `"safari"`, Apple Safari, on Mac OS,
+* `"safari_ios"`, Apple Safari, on iOS,
+* `"servo"`, the experimental Mozilla engine.
+
+No value is mandatory.
+
+Each of these properties contains a `support-statement` object with the
+practical compatibility information for this sub-feature and this browser.
+
+### The `support-statement` object
+This object is the key element of each browser compat information. It is a
+support-statement object. It is an array of `support` objects, but if there
```

Comment:
No, it is an array of `simple_support-statement` objects (this is important because we already have an object whose name is "support", and it is not this one).